apps/src/componentLibrary/dropdown/checkboxDropdown/CheckboxDropdown.tsx

```diff
@@ -57,6 +56,12 @@ export interface CheckboxDropdownProps
       | React.MouseEvent<HTMLButtonElement>
       | React.MouseEvent<HTMLAnchorElement>
   ) => void;
+  /** CheckboxDropdown selectAllText
+   * Text for "Select All" button */
+  selectAllText: string;
```

Comment:
I could also make these optional and include English defaults, but I thought that might produce undesirable behavior (ie, not including translations for these components).

Reply:
No longer relevant in latest revision, as the consumer is required to supply the strings if they wish to use this component.